sdk/python/tests/compiler/testdata/parallelfor_item_argument_resolving.yaml

```diff
@@ -27,36 +27,24 @@ spec:
               _parser = argparse.ArgumentParser(prog='Consume', description='')
               _parser.add_argument("--param1", dest="param1", type=str, required=True, default=argparse.SUPPRESS)
               _parsed_args = vars(_parser.parse_args())
-              _output_files = _parsed_args.pop("_output_paths", [])
               
               _outputs = consume(**_parsed_args)
-
-              _output_serializers = [
-
-              ]
-
-              import os
-              for idx, output_file in enumerate(_output_files):
-                  try:
-                      os.makedirs(os.path.dirname(output_file))
-                  except OSError:
-                      pass
-                  with open(output_file, 'w') as f:
-                      f.write(_output_serializers[idx](_outputs[idx]))
-        image: "tensorflow/tensorflow:1.13.2-py3"
+        image: "python:3.7"
       inputs: 
         parameters: 
           - 
-            name: produce-list-of-strings-output
+            name: produce-list-of-strings-Output
       metadata:
-        annotations:
+        labels:
+          pipelines.kubeflow.org/pipeline-sdk-type: kfp
```

Comment:
nit: are these labels telemetry purpose?

Reply:
Hmm. Yes. I'll remove it.